docs/source/roles/system_manager/manage_deployments.md

```diff
@@ -6,6 +6,22 @@
 This document assumes that you already have access to a {ref}`Safe Haven Management (SHM) environment <deploy_shm>` and one or more {ref}`Secure Research Environments (SREs) <deploy_sre>` that are linked to it.
 ```
 
+```{include} ../../deployment/snippets/00_symbols.partial.md
+:relative-images:
+```
+
+(renew_ssl)=
+
+## {{alarm_clock}} Renewing SRE Domain Certificates
+
+SREs will need to periodically have their SSL certificates renewed so that the SRE URL can be accessed with HTTPS. After each 90 day period that the SRE is live, re-run the script to update the certificate.
+
+![Powershell: five minutes](https://img.shields.io/static/v1?style=for-the-badge&logo=powershell&label=local&color=blue&message=ten%20minutes) at {{file_folder}} `./deployment/secure_research_environment/setup`
+
+```powershell
+PS> ./Update_SRE_SSL_Certificate.ps1 -shmId <SHM ID> -sreId <SRE ID>
```

Comment:
We should explain `<SHM ID>` and `<SRE ID>` like in the other sections https://data-safe-haven.readthedocs.io/en/develop/roles/system_manager/manage_deployments.html